languages/rust/oso/Cargo.toml

```diff
@@ -41,14 +41,18 @@ rustyline = { version = "6.3.0", optional = true }
 rustyline-derive = { version = "0.3.1", optional = true }
 lazy_static = "1.4.0"
 
+uuid = { version = "0.6.5", optional = true }
+
 [dev-dependencies]
 anyhow = "1.0.33"
 criterion = "0.3.3"
 oso-derive = { path = "../oso-derive", version = "=0.8.2" }
 tempfile = "3.1.0"
 static_assertions = "1.1.0"
+uuid = "0.6.5"
 
 [features]
 default = ["derive"]
 cli = ["rustyline", "rustyline-derive", "anyhow"]
 derive = ["oso-derive"]
+extra = ["uuid"]
```

Comment:
As a conditional dependency, you would still be able to enable it with `oso = { version = ..., features = ["uuid"]}`. I think the `extra` feature would group all similar dependencies.

Reply:
@samscott89 is right.  I do think @gj has a good point though.  I didn't honestly know what would be the best name for the group, so just left it as `extra`, as I've seen that used for `uuid` features in other crates.  But, something more appropriate might be warrented - `polarclass` or something referencing that these would be features that provide implementations for `PolarClass` on foreign types?

Reply:
Given the below idea of calling the feature `uuid-06` (or similar), it might not make sense to group all optional impls together. Not suggesting this for now, but maybe eventually a `diesel` group that contains `uuid-06` and whatever other optional impls (if any) are required for diesel.